packages/nvidia_nat_core/src/nat/llm/huggingface_llm.py

```
@@ -145,7 +145,7 @@ async def huggingface_provider(
     except ImportError:
         raise ValueError("HuggingFace dependencies not installed. \n"
                          "Install with:\n"
-                         "  `pip install \"transformers[torch,accelerate]~=4.57\"")
+                         "  `pip install \"transformers[torch,accelerate]>=5.0,<6.0\"")
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)